LobbyCompatibility/Patches/LeaderboardNamePatch.cs

```diff
@@ -1,26 +1,17 @@
-﻿using System.Reflection;
 using HarmonyLib;
 using Steamworks;
 
 namespace LobbyCompatibility.Patches;
 
 [HarmonyPatch]
-[HarmonyPriority(Priority.First)]
+[HarmonyPriority(Priority.Last)]
 [HarmonyWrapSafe]
 internal class LeaderboardNamePatch
 {
-    [HarmonyTargetMethod]
-    private static MethodBase TargetMethod()
-    {
-        return typeof(SteamUserStats).Assembly.GetType("Steamworks.ISteamUserStats")!.GetMethod(
-            "FindOrCreateLeaderboard", (BindingFlags)60)!;
-    }
-
+    [HarmonyPatch(typeof(ISteamUserStats), nameof(ISteamUserStats.FindOrCreateLeaderboard))]
```

Comment:
For consistency with other patches, I'd probably rename the class to the harmony target class, and add [HarmonyPatch(...)] to the class, with the method HarmonyPatch to the method.